cli/js/index.js

```diff
@@ -1,6 +1,53 @@
 Error.stackTraceLimit = 99; // rust stack traces can get pretty big, increase the default
 
-import { NodeConfig, spawnNode } from "lumina-node";
+import { AppVersion, Blob, Namespace, NodeConfig, TxClient, protoEncodeSignDoc, spawnNode } from "lumina-node";
+import { secp256k1 } from "@noble/curves/secp256k1";
+import { Registry } from "@cosmjs/proto-signing";
+
+
+// Expose classes on window so they can be used from the console
+window.AppVersion = AppVersion;
+window.Blob = Blob;
+window.Namespace = Namespace;
+
+// cat ci/credentials/bridge-0.address
+window.bridge0Address = "celestia1t52q7uqgnjfzdh3wx5m5phvma3umrq8k6tq2p9";
```

Comment:
I'm not sure I like the kitchen-sink approach of adding a test transaction submission to our local node here. 
If it's not too much work could you add a separate UI box with idk, IP, address, and key (+ maybe amount) plus a button to do a submission? My worry here, apart from abstract code aesthetics, is that after few such add-ons it'll become a chore to trace what is doing what and why.

Reply:
I don't like this either, and I don't think it should be there at all. However I'm not sure if I'd like to improve on this too :upside_down_face:. I'd like to put work in having a [javascript tests with cypress](https://github.com/eigerco/lumina/issues/483) instead.

On the other hand I can't promise we will do the cypress testing soon, even if it is very important for me to do so, so if you insist I can do the suggestion.

Reply:
I think if there's a plan to wean off of this approach this is fine to keep for now.